src/pages/index.tsx

```diff
@@ -215,3 +218,10 @@ export default ({ data }: PageProps<GatsbyTypes.HomePageQuery>) => {
     </Layout>
   );
 };
+
+export default withPrismicPreview(IndexPage, [
+  {
+    repositoryName: "sheltertech",
+    linkResolver,
+  },
+]);
```

Comment:
It's kind of annoying that we need to repeat these arguments for each page. Could we write our own HOC that wraps `withPrismicPreview` by providing the `repositoryName` and `linkResolver` options? It seems that the API they provide allows you to have one app connected to multiple Prismic sites, but since I don't see us needing to support mulitple Prismic sites, I think it'll be much simpler to have a HOC that provides the one Prismic site config options.